packages/mui-joy/src/Box/Box.test.js

```diff
@@ -23,19 +23,28 @@ describe('Joy <Box />', () => {
     refInstanceof: window.HTMLDivElement,
   }));
 
-  it('respects theme from context', () => {
-    const { container } = render(
-      <ThemeProvider
```

Comment:
Changed to `CssVarsProvider`, when would we use `ThemeProvider` with Joy UI? It's not clear to me why the test was added in the first place.

Reply:
Since this PR, I think `ThemeProvider` is meant to be used for writing internal tests that related to theming.

From the consumer perspective, they don't need to use it.

Reply:
But shouldn't we rather test using the API that the consumers would use? I wouldn't even export the `ThemeProvider`. It should be considered internal component that's used in the `CssVarsProvider`.